### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by gargarchit**
Hello
I here Updated ReadMe therefore it will ook much better and easier to read
please merge
**Thank You**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,14 @@
-# About this Repo
+<h1 align = left> About this Repo </h1>
```
Comment (gszr) [inline README.md]: This seems... unnecessary?

File: README.md [hunk 2]
```diff
@@ -1,9 +1,14 @@
...
 
-This is the Git repo of the Docker [official image](https://docs.docker.com/docker-hub/official_repos/) for [kong](https://registry.hub.docker.com/_/kong/). See [the Docker Hub page](https://registry.hub.docker.com/_/kong/) for the full readme on how to use this Docker image and for information regarding contributing and issues.
+------
```
Comment (gszr) [inline README.md]: ditto

Comment (gargarchit) [reply README.md]: This is to add Horizontal line

**Comment by gargarchit** [comment]
Thank You for merging this

**Comment by thibaultcha** [comment]
As written in the README, this file is to be auto-generated by https://github.com/docker-library/docs/blob/master/generate-repo-stub-readme.sh

**Pull request merged**